contact.html

```diff
@@ -72,31 +73,31 @@ <h1 class="description__title">Want to talk with us?</h1>
           action="https://formspree.io/f/mayvnjln"
           method="POST"
         >
-          <label class="form__label" for="Name">Name:</label>
+          <label class="form__label" for="Name">Your Name:</label>
           <input
             class="form__input"
             type="text"
             name="Name"
             id="name"
             aria-required="true"
             required
-            placeholder="e.g John Doe"
+            placeholder="e.g   Sasha Smith (they/them)"
             autocomplete="name"
           />
```

Comment:
I think it'd be nice if we included pronouns and different names to our forms so I've added this

Reply:
ohh, Got it!